hit-testing-explainer.md

```diff
@@ -128,18 +156,19 @@ This is a partial IDL and is considered additive to the core IDL found in the ma
 //
 partial interface XRSession {
   Promise<XRHitTestSource> requestHitTestSource(XRHitTestOptionsInit options);
-  Promise<XRHitTestSource> requestHitTestSourceForTransientInput(XRTransientInputHitTestOptionsInit options);
+  Promise<XRTransientInputHitTestSource> requestHitTestSourceForTransientInput(XRTransientInputHitTestOptionsInit options);
```

Comment:
`XRTransientInputHitTestOptionsInit` should be defined somewhere

Reply:
They are at line 149 (before PR) / 178 (after PR).